frontend/src/components/common/SortTablePopover.tsx

```diff
@@ -44,11 +44,9 @@ const SortTablePopover = ({
           <>
             <PopoverTrigger>
               <Button
-                size="sm"
-                pl="10%"
-                ml="0"
-                rightIcon={<FilterOptionsIcon />}
-                bg="#blue.300"
+                minWidth="10%"
+                leftIcon={<FilterOptionsIcon />}
```

Comment:
not a big deal, but was wondering if we want to use the icon that matches the one in Figma? i can't find it in the eva icons package tho

Reply:
as per design team, we don’t have the icon in the design, so this is the one they wanted to use instead!